lib/result-view.coffee

```diff
@@ -63,13 +63,27 @@ class ResultView
                 editor.insertText(bubbleText)
         actionPanel.appendChild(openButton)
 
+        insertButton = document.createElement('div')
+        insertButton.classList.add 'action-button',
+            'insert-button', 'icon', 'icon-triangle-down'
+        insertButton.onclick = =>
+            bubbleText = @getAllText()
+            editor = atom.workspace.getActiveTextEditor()
+            editor.insertNewlineBelow()
+            editor.insertText('result: \n' + bubbleText + '\n',
+              select: true)
+            editor.toggleLineCommentsInSelection()
+        actionPanel.appendChild(insertButton)
```

Comment:
This functionality doesn't really make sense inside the watch panel.
```coffeescript
if @marker?
    actionPanel.appendChild(insertButton)
```
This should (haven't tested this yet) hide the button if in the watch sidebar.